### DISCUSSION TITLE
How to disable progress bar which rolled out in latest updates?

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#999] [Q&A]
Hi, I like the idea of the progress bar when transcribing, however it takes a bit longer to do the transcription now, how can i disable it?

**Comment by glangford** [comment]
`--verbose True` disables it.

**Reply by Abdurrafey-Siddiqui** [reply]
True but then it has to form the timestamps and their corresponding words then output it into the CLI which also takes long. Was wondering if it's possible to go back to the old one where it doesn't output anything, that way it was much faster

**Comment by juanmc2005** [comment]
Hello, I am also looking for a way to ignore all console outputs as they interfere with the way my application works.
Is there a way to remove this or should I go back to a previous version?

**Reply by jongwook** [reply]
It's not currently supported in the command line tool `whisper`, but in Python you can specify `verbose=None` which will disable the console output.
